Packages/DV3D/StructuredGridPlot.py

```diff
@@ -396,11 +396,11 @@ def buildBaseMap(self):
         self.mapManager = MapManager( roi = self.roi )
         self.renderer.AddActor( self.mapManager.getBaseMapActor() )
 
-#         if self.baseMapActor <> None: self.renderer.RemoveActor( self.baseMapActor )
-#         world_map =  None
-#         map_border_size = 20
-#
-#         self.y0 = -90.0
+#         if self.baseMapActor <> None: self.renderer.RemoveActor( self.baseMapActor )               
```

Comment:
Yes I did source the setup_for_development.sh script, but I commit with the "-n" option- otherwise I get pages of these "trailing whitespace errors", and often most of them are in someone else's code.

From: Charles Doutriaux <notifications@github.com<mailto:notifications@github.com>>
Reply-To: UV-CDAT/uvcdat <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Monday, March 2, 2015 5:44 PM
To: UV-CDAT/uvcdat <uvcdat@noreply.github.com<mailto:uvcdat@noreply.github.com>>
Cc: "Maxwell, Thomas P. (GSFC-606.2)[SCIENCE APPLICATIONS INTL CORP]" <thomas.maxwell@nasa.gov<mailto:thomas.maxwell@nasa.gov>>
Subject: Re: [uvcdat] Vcs3 d hide 3d controls for 2d plots (#1095)


In Packages/DV3D/StructuredGridPlot.py<https://github.com/UV-CDAT/uvcdat/pull/1095#discussion_r25646529>:

> @@ -396,11 +396,11 @@ def buildBaseMap(self):
>          self.mapManager = MapManager( roi = self.roi )
>          self.renderer.AddActor( self.mapManager.getBaseMapActor() )
>
> -#         if self.baseMapActor <> None: self.renderer.RemoveActor( self.baseMapActor )
> -#         world_map =  None
> -#         map_border_size = 20
> -#
> -#         self.y0 = -90.0
> +#         if self.baseMapActor <> None: self.renderer.RemoveActor( self.baseMapActor )


did you source "setup_for_development.sh" in "Scripts" directory? It shouldn't let you commit with trailing spaces

—
Reply to this email directly or view it on GitHub<https://github.com/UV-CDAT/uvcdat/pull/1095/files#r25646529>.